targets/simple_switch/tests/test_packet_redirect.cpp

```diff
@@ -35,7 +35,7 @@
 
 #include "simple_switch.h"
 
-#include "utils.h"
+#include "../../test_utils/utils.h"
```

Comment:
Add `test_utils` as an "include dir" by modifying `AM_CPPFLAGS` in the Makefile, to avoid the ugly import.

Reply:
Ok